### PR TITLE
Added Porter Mixins and Plugins details in Readme.md

### LINKED ISSUE #1624: Adding More Details about mixins and plugins with supported platform in README.md
**What design is being proposed?**
In README.md we should add some details about the porter mixins and plugins, and what all platform porter supports, this will help contributors to get basic insights from the readme.md, if they want to learn more about them they should read docs.

**This can be performed by adding tables like below [Source: Meshery(Layer5)]** 

design1
design2

**Additional Context**

**Risks/Concerns**

**What other approaches were considered?**

**Implementation details**

# Checklist
- [ ] An announcement of this proposal has been sent to the Porter mailing list: https://porter.sh/mailing-list
- [ ] This proposal has remained open for at least one week, to allow time for community feedback.

### PR DESCRIPTION
**Pull request by DARK-art108**
# What does this change
Added all the available Porter Mixins and Plugins in a Tabular format in Readme.md

# What issue does it fix
Closes #1624 

[1]: https://porter.sh/src/CONTRIBUTING.md#when-to-open-a-pull-request

# Notes for the reviewer
_Put any questions or notes for the reviewer here._

# Checklist
- [ ] Unit Tests
- [x] Documentation
- [ ] Schema (porter.yaml)

If this is your first pull request, please add your name to the bottom of our [Contributors][contributors] list. Thank you for making Porter better! 🙇‍♀️

[contributors]: https://porter.sh/src/CONTRIBUTORS.md

### CODE REVIEW

**Comment by DARK-art108** [comment]
@carolynvs Please take a look at it! :)

File: README.md [hunk 1]
```diff
@@ -13,6 +13,37 @@ application.
...
+
+[Porter Mixins](https://porter.sh/mixins/) are available for below platform's:
+
+| Platform                                                                                                                                                                                                                        | Supported?  |
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :---------: |
+| <img src="docs\static\images\mixins\docker_icon.png" width="20" height="20" vertical-align="middle" /> [Docker](https://porter.sh/mixins/docker/)                                            |     ✔️      |
+| &nbsp;&nbsp;&nbsp; <img src="docs\static\images\mixins\docker-compose.png" width="20" height="20" vertical-align="middle" /> [Docker-Compose](https://porter.sh/mixins/docker-compose/)            |     ✔️      |
```
Comment (carolynvs) [inline README.md:25]: Looks like you forgot the leading whitespace on the first row in the table (for Docker and hashicorp below). Can you either remove the leading whitespace on the other rows, or add the same whitespace to the first row of each table?

**Review by carolynvs — CHANGES_REQUESTED** [review]
Looks great!

File: README.md (hunk 1 above)
Comment (DARK-art108) [reply README.md:25]: Hello @carolynvs, I am not getting _leading whitespaces_, can you be more clear!

Comment (carolynvs) [reply README.md:25]: On line 24, in the first table cell you have this

`<img src="docs\static\images\mixins\docker_icon.png" ...`

On line 25 (and the remaining cells in the table), you have the HTML whitespace character `&nbsp;` (stands for non-breaking space) in front of the image, which shifts the image over to the right when the page is displayed.

`&nbsp;&nbsp;&nbsp; <img src="docs\static\images\mixins\docker-compose.png"`

If those whitespace characters `&nbsp;` were unintentional, would you please remove them from both tables?

Comment (DARK-art108) [reply README.md]: @carolynvs I have pushed the changes do take a look at it!

**Review by carolynvs — APPROVED** [review]
This is a really nice addition to our readme. Thanks!

**Pull request merged**